src/DynamoUtilities/DynamoPathManager.cs

```diff
@@ -74,16 +69,16 @@ public IEnumerable<string> PreloadLibraries
         /// <summary>
         /// The Logs folder.
         /// </summary>
-        public string Logs { get; set; }
+        public string Logs { get; private set; }
 
         /// <summary>
         /// The Dynamo folder in AppData
         /// </summary>
-        public string AppData { get; set;}
+        public string AppData { get; private set;}
 
-        public string GeometryFactory { get; set; }
+        public string GeometryFactory { get; private set; }
 
-        public string AsmPreloader { get; set; }
+        public string AsmPreloader { get; private set; }
```

Comment:
Why not `public readonly`?  Will these be changed in the lifetime of the `DynamoPathManager`?